portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintruleset.js

```diff
@@ -472,11 +472,17 @@ var LinearInterpRuleSet = (function () {
 	this.value_key = params.value_key;
 	this.value_range = params.value_range;
 	this.log_scale = params.log_scale; // boolean
-	this.inferred_value_range;
+
+	this.rangeTypes = {
+		'SIGNED': 'SIGNED',
+		'UNSIGNED': 'UNSIGNED',
+		'NON_POSITIVE': 'NON_POSITIVE'
```

Comment:
maybe @rnugraha means that if the range includes 0, its non-positive, as opposed to negative.

in any case I think the naming is a bit confusing, mixing the "signed" language with "non_positive". Maybe just positive, negative, both, with 0's inclusion being implied?